libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -559,7 +583,7 @@ void AP_RCProtocol_CRSF::start_uart()
     _uart->set_flow_control(AP_HAL::UARTDriver::FLOW_CONTROL_DISABLE);
     _uart->set_blocking_writes(false);
     _uart->set_options(_uart->get_options() & ~AP_HAL::UARTDriver::OPTION_RXINV);
-    _uart->begin(CRSF_BAUDRATE, 128, 128);
+    _uart->begin(CRSF_BAUDRATE, 256, 256);
```

Comment:
mhmm, so this turns out to be useless? we have always had 512+512

Reply:
Yes removed